go-controller/pkg/ovn/policy.go

```diff
@@ -257,30 +262,56 @@ func (oc *Controller) createDefaultDenyPGAndACLs(namespace, policy string, nsInf
 	return nil
 }
 
-func (oc *Controller) setACLLoggingForNamespace(ns string, namespacePolicies map[string]*networkPolicy, portGroupIngressDenyName string, portGroupEgressDenyName string, logSeverityConfig ACLLoggingLevels) error {
+func (oc *Controller) updateACLLoggingForPolicy(np *networkPolicy, logLevel string) error {
+	np.Lock()
+	defer np.Unlock()
+
+	if !np.created {
+		return NetworkPolicyNotCreated
+	}
+	var ovsDBOps []ovsdb.Operation
+	acls := oc.buildNetworkPolicyACLs(np, logLevel)
+	ops, err := libovsdbops.UpdateACLsLoggingOps(oc.nbClient, ovsDBOps, acls...)
```

Comment:
This is wierd, you don't need either `ovsDBOps` or `ops`:

`ops, err := libovsdbops.UpdateACLsLoggingOps(oc.nbClient, nil, acls...)`

And then just transact those ops would do.